Astral Drift/Assets/SystemObjects/Scripts/Systems/EnemyWaveSpawner.cs

```diff
@@ -0,0 +1,97 @@
+        {
+            Debug.Log("Out of bounds: Left");
+            float distOutOfBounds = enemy.transform.position.x + halfScreenX;
+            Vector2 newPos = new Vector2((distOutOfBounds * -1) + enemy.transform.localScale.x / 2, 0);
+            enemy.transform.position += (Vector3)newPos;
+        }else if (enemy.transform.position.x > halfScreenX)
+        {
+            Debug.Log("Out of bounds: Right");
+            float distOutOfBounds = halfScreenX - enemy.transform.position.x;
+            Vector2 newPos = new Vector2(distOutOfBounds - enemy.transform.localScale.x / 2, 0);
+            enemy.transform.position += (Vector3)newPos;
+        }
+
+        /*for (int a = difficultyManager.lastTopIndex; a < difficultyManager.enemyDataList.Count; a++)
+        {
+            if (Mathf.Abs(enemy.transform.position.x - enemies[a].transform.position.x) < maxEnemyCollisionRange && Mathf.Abs(enemy.transform.position.y - enemies[a].transform.position.y) < maxEnemyCollisionRange)
+            {
+            }
+        }*/
+    }
+}
+
+/*
+switch (formation)
+        {
+            case Enumerators.EnemyFormationTypes.HorizontalLine:
+                for (int n = 0; n < enemyFormation.Amount; n++)
+                {
+                    spawnedEnemies.Add(Instantiate(enemyFormation.EnemyPrefab, enemyFormation.EnemyPosition + new Vector2(distBetweenEnemies * n, 0), Quaternion.identity));
+                }
+                break;
+            case Enumerators.EnemyFormationTypes.VerticalLine:
+                for (int n = 0; n < enemyFormation.Amount; n++)
+                {
+                    spawnedEnemies.Add(Instantiate(enemyFormation.EnemyPrefab, enemyFormation.EnemyPosition + new Vector2(0, distBetweenEnemies * n), Quaternion.identity));
+                }
+                break;
+            case Enumerators.EnemyFormationTypes.RightDiagonal:
+                for (int n = 0; n < enemyFormation.Amount; n++)
+                {
+                    spawnedEnemies.Add(Instantiate(enemyFormation.EnemyPrefab, enemyFormation.EnemyPosition + new Vector2(distBetweenEnemies * n, distBetweenEnemies * n), Quaternion.identity));
+                }
+                break;
+            case Enumerators.EnemyFormationTypes.LeftDiagonal:
+                for (int n = 0; n < enemyFormation.Amount; n++)
+                {
+                    spawnedEnemies.Add(Instantiate(enemyFormation.EnemyPrefab, enemyFormation.EnemyPosition + new Vector2(distBetweenEnemies * n, distBetweenEnemies * n), Quaternion.identity));
+                }
+                break;
+            case Enumerators.EnemyFormationTypes.VFormation:
+                for (int n = (int)-halfAmount; n < halfAmount + 1; n++)
+                {
+                    spawnedEnemies.Add(Instantiate(enemyFormation.EnemyPrefab, enemyFormation.EnemyPosition + new Vector2(halfAmount + n, Mathf.Pow(n, 2)), Quaternion.identity));
+                }
+                break;
```

Comment:
Zijn deze comments nog ergens voor nodig? Zo niet, weghalen!

Reply:
noh, it gone